modules/monitoring-specifying-how-a-service-is-monitored-for-cluster-observability-operator.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// monitoring/cluster-observability-operator/configuring-the-cluster-observability-operator-to-monitor-a-service.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="specifying-how-a-service-is-monitored-for-cluster-observability-operator_{context}"]
+= Specifying how a service is monitored for Cluster Observability Operator
+
+To use the metrics exposed by the sample service you created in the "Creating a `MonitoringStack` object for the Cluster Observability Operator" section, you must configure monitoring components to scrape metrics from the `/metrics` endpoint. 
```

Comment:
Changed the reference, but unfortunately it's not possible to add a hyperlink because of restrictions in the OCP docs repo.

Reply:
Suggestion: maybe put something like "See [x] in the "Additional resources" section" and then add an .Additional resources after the module? IDK if this is helpful, though

Reply:
To me, it seems very awkward to do that because all of the sections are on this page. Adding Additional resources section with a link just adds complexity and more mental friction, imo.

Reply:
I think it helps that I include the details about what got configured in the other section in the list of prereqs for this procedure.